src/commands/graph/graph-pull.js

```diff
@@ -70,30 +99,61 @@ const graphPull = async (options, command) => {
     sessionId: oneGraphSessionId,
   })
 
+  if (!next) {
+    return
+  }
+
   if (next.errors) {
     error(`Failed to fetch Netlify Graph cli session events: ${JSON.stringify(next.errors, null, 2)}`)
   }
 
   if (next.events) {
-    const ackIds = []
+    const ackEventIds = []
     for (const event of next.events) {
-      await handleCliSessionEvent({
-        netlifyToken,
-        event,
-        netlifyGraphConfig,
-        schema,
-        sessionId: oneGraphSessionId,
-        siteId: site.id,
-        siteRoot: site.root,
-      })
-      ackIds.push(event.id)
+      try {
+        const audience = event.audience || OneGraphClient.eventAudience(event)
+
+        if (audience === 'CLI') {
+          const eventName = OneGraphClient.friendlyEventName(event)
+          log(`${chalk.magenta('Handling')} Netlify Graph: ${eventName}...`)
+          const nextSchemaId = readSchemaIdFromLockfile({ siteRoot: site.root })
+
+          if (!nextSchemaId) {
+            warn('Unable to load schemaId from Netlify Graph lockfile, run graph:pull to update')
```

Comment:
more places where we tell them to run `graph:pull` when they're running `graph:pull`